zb-db/src/main/java/io/zeebe/db/impl/rocksdb/RocksDbConfiguration.java

```diff
@@ -13,10 +13,18 @@
 
   public static final long DEFAULT_MEMORY_LIMIT = 512 * 1024 * 1024L;
   public static final int DEFAULT_UNLIMITED_MAX_OPEN_FILES = -1;
+  public static final int DEFAULT_MAX_WRITE_BUFFER_NUMBER = 6;
+  public static final int DEFAULT_MIN_WRITE_BUFFER_NUMBER_TO_MERGE = 3;
+  public static final boolean DEFAULT_STATISTICS_ENABLED = false;
+  public static final boolean DEFAULT_WAL_DISABLED = false;
+  public static final int DEFAULT_IO_RATE_BYTES_PER_SECOND = 0;
```

Comment:
This is now different have you benchmarked this?

Reply:
What is different from the previous settings? I thought I kept the exact same ones :thinking: We had 6 memtables, and we would merge `Math.min(3, 6)` so 3, and statistics and WAL was disabled.

Reply:
I meant the io rate :D

Reply:
By default it's disabled, meaning there is no limit, which is the same as before. The difference I guess is removing `bytesPerSync`, but like I said I found it had no impact whatsoever - you can see `np-no-fsync` where the only difference is removing this param (and nothing else), and it's pretty much the same as baseline.